cmd/namespace/namespace.go

```diff
@@ -64,3 +71,34 @@ func Namespace(ctx context.Context) *cobra.Command {
 	cmd.AddCommand(Wake(ctx))
 	return cmd
 }
+
+type fakeK8sProvider struct {
```

Comment:
can we move this to the test file where they are used?

Reply:
+1 @rakshitgondwal let us know if you can move the `fakeK8sProvider` to the test file, if not we will move it and merge the PR

Reply:
Oh yes, so sorry, just busy with other stuff, will fix this in 1-2 days.